### DISCUSSION TITLE
Integrate state management

### DISCUSSION THREAD

**Discussion by aolko** [Panchovix/stable-diffusion-webui-reForge#24] [Ideas]
Fix & integrate this extension: https://github.com/SenshiSentou/sd-webui-state-manager
Saves ass when your settings are lost, but last time i installed it in forge it was somewhat lagging the whole UI, please check. However, when it comes to loading settings forge can't deal with HIRESFIX_BOTH in img2img...that has no hiresfix, resulting in broken generation.

**Comment by minsky91** [comment]
A1111/Forge's inability to properly maintain and restore last state of the UI is the single most annoying (should I say 'infuriating'? yeah, that's closer to the truth) issue in these platforms for me. PNGInfo doesn't really do much in this regard. PLEASE address this in the nearest update of ReForge. Don't really care whether  via an extension or a built-in feature, as long as it frees me up from the necessity of recalling and restoring the last 10-someting settings I used yesterday.  Will switch to reForge in a blink of an eye, once this is implemented.
